virtual-organizations/IceCube.yaml

```diff
@@ -58,13 +58,24 @@ DataFederations:
         AllowedOrigins:
           - IceCube_StashCache_origin
           
-      - Path: /icecube/NRP-UNL/PROTECTED
+      - Path: /nrpnb/protected/icecube/
         Authorizations:
           - SciTokens:
               Issuer: https://chtc.cs.wisc.edu/icecube
-              Base Path: /icecube/NRP-UNL/PROTECTED
+              Base Path: /nrpnb/protected/icecube/
               Map Subject: False
         AllowedOrigins:
           - NEBRASKA_NRP_OSDF_ORIGIN
         AllowedCaches:
           - ANY
+          
+      - Path: /mghpcc/protected/icecube/
+        Authorizations:
+          - SciTokens:
+              Issuer: https://chtc.cs.wisc.edu/icecube
+              Base Path: /mghpcc/protected/icecube/
```

Comment:
Similar to above -- make the base component of the namespace the VO not the origin name.

Reply:
sure, solved!